Formula/cfitsio.rb

```diff
@@ -39,4 +40,18 @@ def install
       end
     end
   end
+
+  test do
+    cp Dir["#{pkgshare}/testprog/testprog*"], testpath
+    flags = (ENV.cflags || "").split + (ENV.cppflags || "").split + (ENV.ldflags || "").split
```

Comment:
Is this flag propagation part necessary? Won't the compiler pick these up from the existing environment variables? I'm wondering because I think the flags variables, especially the LDFLAGS, might have embedded spaces & quotes which `split` may not handle correctly.